### ISSUE TITLE
Release Helm RPM in Kubernetes Repo

### ISSUE THREAD

**Issue by andremarianiello** [kubernetes/helm#2773]
Helm would be easier to install if it was provided by the kubernetes rpm repo with kubeadm, kubectl, etc. At the very least, a Helm RPM should be available somewhere.

**Comment by bacongobbler** [comment]
related: #2616

**Comment by bacongobbler** [comment]
Giving the same response as in #2616:

So as far as I remember what we discussed in the developer call, we are not going to move forward with hosting "package-manager-specific" code in Helm core. We cannot keep up with the maintenance of an endlessly sprawling number of ways to install Helm. However, third party package managers are more than welcome to create repositories at [kubernetes-helm](https://github.com/kubernetes-helm) as part of the community shift.

With that in place, the story moving forward would be:

a) The community will be responsible for supporting the infrastructure of their own package manager
b) Helm Core Maintainers can create repositories at https://github.com/kubernetes-helm for hosting these package manager-specific code repositories (upon request/approval)
c) We will continue to release Helm as necessary, circling in package manager maintainers into upcoming Helm releases through the roadmap, emails, weekly Helm dev calls etc.

TL;DR it's up to the community to maintain these third party packages.

Thanks!

**Comment by jamshid** [comment]
It's funny that the package manager for kubernetes is not available to package managers for linux as a deb or rpm? 
Hopefully someone in the community does set that up. Some people and places frown on unmanaged linux binary installs.
FWIW it is available on snap:
https://snapcraft.io/install/helm/ubuntu
which seems to be a cross-linux package manager:
https://snapcraft.io/docs/installing-snap-on-centos

**Comment by krc** [comment]
Staggering incompetence, well done. Just what I expect from wider k8s community at this point though, so good work maintaining the standard...
